src/search.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * @copyright Copyright (c) 2024 Fon E. Noel NFEBE <me@nfebe.com>
+ *
+ * @author Vincent Petry <me@nfebe.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import Vue from 'vue'
+
+import { getRequestToken } from '@nextcloud/auth'
+import { emit } from '@nextcloud/event-bus'
+import { translate, translatePlural } from '@nextcloud/l10n'
+import { generateFilePath } from '@nextcloud/router'
+
+import RoomSelector from './components/RoomSelector.vue'
+
+import { fetchConversation } from './services/conversationsService.js'
+
+import '@nextcloud/dialogs/style.css'
+
+(function(OC, OCA, t, n) {
+
+	/**
+	 *
+	 */
+	function init() {
+		if (!OCA.Core) {
+			return
+		}
+		console.debug('Initializing unified search plugin-filters from talk')
+		OCA.Core.UnifiedSearch.registerFilterAction({
```

Comment:
OCA is for apps, Core is not apps. I didn't check recent JS changes in servers, but should this be OCP.UnifiedSearch?